src/crafting.cpp

```diff
@@ -50,7 +49,7 @@ void remove_from_component_lookup(recipe* r);
 
 recipe::~recipe()
 {
-    remove_from_component_lookup(this);
+    recipe_dict.remove_recipe( this );
```

Comment:
The word `recipe` appears twice, sounds a bit redundant. One would expect a `remove` function, called on a `recipe_dict` to remove ... well ... a recipe, what else? The same applies to the `add_recipe` and similar functions. They operate on a `recipe_dict`, which pretty much indicates what they add/find/remove.